src/main/webapp/index.jsp

```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <title>NeighborHelp</title>
+    <link rel="stylesheet" href="homepage_style.css">
+    <script src="homepage_script.js"></script>
+    <script src="https://kit.fontawesome.com/71105f4105.js" crossorigin="anonymous"></script> 
+  </head>
+  <body>
+      <!--Site Header-->
+      <header>
+          <nav>
+              <div id="userpage-icon">
+                  <a href="user_profile.html">
+                      <i class="fas fa-user-circle fa-3x" title="Go to User Page"></i>
+                  </a>
+              </div>
+          </nav>
+          <div id="title">
+              NeighborHelp
+          </div>
```

Comment:
For the title, I would recommend using an `<h1>` element for that.

Reply:
That's probably better, I'll change this!